src/App.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
+import {Route} from 'react-router-dom';
 
 const App = () => (
   <div>
-    <Navigation />
+    <Route exact path="/" component ={Home}/>
+    <Route exact path= "/Navigation" component ={Navigation}/>
```

Comment:
Babatunde, Navigation component should not be within Route since that component you want to 'show' at all times.